src/objects/game.ts

```diff
@@ -1,8 +1,7 @@
-import { saveGameState } from "../utilities/localStorage";
-import { Business } from "./business";
+import eventCenter, { ClockEvents, GameplayEvents } from "../events/eventCenter";
```

Comment:
I collapsed the business and clock object down to this (really huge) file. Because we were instantiating a new business object and a new clock object, saving state to local storage periodically got real funky. Now everything happens from this Game object, and there is a bit more consistency (although still a race condition on saving state and then getting state back out of local storage again that needs to be solved for)